intelmq/bots/parsers/alienvault/parser_otx.py

```diff
@@ -46,16 +65,38 @@ def process(self):
                               indicator["indicator"])
                 # URLs
                 elif indicator["type"] in ['URL', 'URI']:
-                    event.add('source.url',
-                              indicator["indicator"])
+                    resource = indicator["indicator"] \
+                        if '://' in indicator["indicator"] \
+                        else 'http://' + indicator["indicator"]
+                    event.add('source.url', resource)
                 # CIDR
                 elif indicator["type"] in ['CIDR']:
                     event.add('source.network',
                               indicator["indicator"])
-                # FilePath, Mutex, CVE - TODO: process these IoCs as well
+
+                # CVE
+                elif indicator["type"] in ['CVE']:
+                    additional_indicator['CVE'] = indicator["indicator"]
+                    # TODO: Process these IoCs: FilePath, Mutex
                 else:
                     continue
 
+                if 'tags' in pulse:
+                    additional_indicator['tags'] = pulse['tags']
+                if 'modified' in pulse:
+                    additional_indicator['time_updated'] = \
+                        pulse["modified"][:-4] + "+00:00"
+                if 'industries' in pulse:
+                    additional_indicator['industries'] = pulse["industries"]
+                if '' in pulse:
```

Comment:
i missed that part where i had already added stuff for targeted_countries. Removed it, the new if is there to add these. :|